Cdk/cdk.json

```diff
@@ -0,0 +1,24 @@
+{
+  "app": "npx ts-node --prefer-ts-exts bin/aws-ec2-imagebuilder-cdk-example.ts",
+  "context": {
+    "buildCompletionNotificationEmails": ["AlejandroRosalez@example.com"],
```

Comment:
Perhaps some other wording around this, eg `"< insert your email here >"` and associated updates in the README about it?

Reply:
1. the sample email address is from https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/safenames.html#asns which is standard for all open-source, 
2. it would be better if user don't need to change anything for first deployment.
I will update the readme to mention the `AlejandroRosalez@example.com` is just a sample email. Let me know how do you think.

Reply:
No issues on this.